### PR TITLE
feat: add DefaultShader field to set the default shader instead of the default of each RP

### PR DESCRIPTION
**Pull request by schinkowski**

### CODE REVIEW

**Review by seandillon92 — COMMENTED** [review]
I think we should document this property in user docs

File: Runtime/Scripts/BaseMeshSync.Textures.cs [hunk 1]
```diff
@@ -146,7 +146,12 @@ partial class BaseMeshSync {
     /// <summary>
     /// Returns the default shader for the active render pipeline.
     /// </summary>
-    private static Shader GetStandardShader() {
+    private Shader GetStandardShader() {
+        if (GetConfigV().GetModelImporterSettings().DefaultShader != null)
```
Comment (sindharta-tanuwijaya) [inline Runtime/Scripts/BaseMeshSync.Textures.cs]: I guess we can cache `GetConfigV().GetModelImporterSettings().DefaultShader` directly into a local variable to reduce function calls.

Comment (schinkowski) [reply Runtime/Scripts/BaseMeshSync.Textures.cs]: Done

**Comment by sindharta-tanuwijaya** [comment]
@schinkowski I did a couple of minor modifications.
Could you please address @seandillon92 comments above ?

> I think we should document this property in user docs

**Comment by schinkowski** [comment]
> @schinkowski I did a couple of minor modifications. Could you please address @seandillon92 comments above ?
> 
> > I think we should document this property in user docs

Done

**Review by seandillon92 — APPROVED** [review]

**Pull request merged**